File: benchmark-java-sync/src/main/java/com/example/benchmarkjavasync/core/model/Application.java
package com.example.benchmarkjavasync.core.model;

import jakarta.persistence.Entity;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;
import java.util.UUID;

public record Application(
        UUID userId,
        String firstName,
        String lastName,
        int passportSerialNumber,
        int passportNumber,
        BigDecimal amount,
        String status) {
}

File: benchmark-kotlin-async/src/main/kotlin/com/example/benchmarkkotlinasync/core/model/Application.kt
package com.example.benchmarkkotlinasync.core.model;

import java.math.BigDecimal;
import java.util.UUID;

data class Application(
        val userId: UUID,
        val firstName: String,
        val lastName: String,
        val passportSerialNumber: Int,
        val passportNumber: Int,
        val amount: BigDecimal,
        val status: String)
